training_toolbox/vehicle_attributes/README.md

```diff
@@ -0,0 +1,93 @@
+# Vehicle attributes recognition: type and color
+
+
+## Train Vehicle Attributes model
+
+To train a Vehicle Attributes model, jump to
+training_toolbox/vehicle_attributes directory. You'll see the folder with a sample code
+demonstrating how to train a VA model.
+
+We provide predefined configuration for: 
+* Vehicle type and color recognition.
+  - Configuration file: [training_toolbox/vehicle_attributes/boxcars116k/config.py](boxcars116k/config.py).
+  - Trained model: [Vehicle Attributes](https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/models/vehicle_attributes/type_color/vehicle-attributes-barrier-0103.tar.gz).
+    A model has two output nodes:
+    . type is 4 components: one for each of 4 types - car, van, truck, bus. The resulting type is argmax of these values.
+    . color is 3 components: LAB format normalized from 0 to 1
+
+For demonstration purpose only a small dataset is uploaded [Boxcars-116k](https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/datasets/boxcars116k.tar.gz).
```

Comment:
The name of the dataset is misleading. Let's change it.

Reply:
@snosov1 changed to cars_100

Reply:
Lena, the new link doesn't work. Is this intended?